.gitmodules

```diff
@@ -46,3 +46,6 @@
 [submodule "third_party/c-dt"]
 	path = third_party/c-dt
 	url = https://github.com/tarantool/c-dt.git
+[submodule "third_party/lua-zlib"]
+	path = third_party/lua-zlib
+	url = https://github.com/tarantool/lua-zlib.git
```

Comment:
We bundle it into our source tarball, so we should put the copyright holder and license into `debian/copyright`.

Reply:
Fixed.